src/pages/contributors/ulli-hafner.adoc

```diff
@@ -0,0 +1,61 @@
+= Ulli Hafner
+:page-name: Ulli Hafner
+:page-linkedin: 
+:page-twitter: 
+:page-github: uhafner
+:page-email: 
+:page-image: avatar/ulli-hafner.png
+:page-pronouns: He/Him/His
+:page-location: Munich, Bavaria, Germany
+:page-firstcommit: 2007
+:page-datepublished: 2024-01-23
+:page-featured: true
+:page-intro: Ulli Hafner is a professor of Software Engineering at the University of Applied Sciences Munich where he advocates for Jenkins in several ways. He is also a software developer, open-source contributor, plugin maintainer, and serves on the Jenkins governance board.
+
+He is also part of the UX SIG, participating in meetings and providing various UX contributions over the years.
+Ulli likes to help new contributors grapple with Jenkins and give talks at Jenkins events, where he can offer his insight into open source.
```

Comment:
Shouldn't this be either 
- "and gives talks at Jenkins events"
- or "and likes to give talks at Jenkins events"?

Reply:
It looks like to me that a "to" is implied here:
```suggestion
Ulli likes to help new contributors grapple with Jenkins and to give talks at Jenkins events, where he can offer his insight into open source.
```

Reply:
I changed the verbiage to more active voicing.
```suggestion
Ulli enjoys helping new contributors grapple with Jenkins and giving talks at Jenkins events, where he can offer his insight into open source.
```